Makefile

```diff
@@ -0,0 +1,17 @@
+CC=g++
+CFLAGS=-Wall -Wextra -Wpedantic -Werror -Iincludes
+SOURCES=debug release
+OBJECTS=*cpp -std=c++17
+
+.PHONY: all
+all: $(SOURCES)
+
+debug: *.cpp includes/*.hpp
+	$(CC) $(OBJECTS) $(CFLAGS) -g -o debug
```

Comment:
Użyj też proszę zmiennych $@ i $< zarówno w celu debug jak i w release
https://stackoverflow.com/questions/3220277/what-do-the-makefile-symbols-and-mean